desktop_version/src/Game.cpp

```diff
@@ -7134,6 +7136,28 @@ void Game::createmenu( enum Menu::MenuName t, bool samemenu/*= false*/ )
         menuxoff = -40;
         menuyoff = 10;
         break;
+    case Menu::language:
+        if (loc::languagelist.size() == 0)
```

Comment:
(Same here, can be changed to `loc::languagelist.empty()`).

Reply:
Changed in 57608cdffdf1656cd93c73370784c66a502aed43.